src/components/v2/V2Project/NewDeployModal.tsx

```diff
@@ -0,0 +1,138 @@
+import { t, Trans } from '@lingui/macro'
+import { Modal } from 'antd'
+import { useDeployProjectPayerTx } from 'hooks/v2/transactor/DeployProjectPayerTx'
+import { useIssueTokensTx } from 'hooks/v2/transactor/IssueTokensTx'
+import { useLocation } from 'react-router-dom'
+import { CheckCircleFilled } from '@ant-design/icons'
+import { CSSProperties, useContext, useState } from 'react'
+import RichButton from 'components/shared/RichButton'
+
+import { ThemeContext } from 'contexts/themeContext'
+import IssueTokenModal from 'components/shared/modals/IssueTokenModal'
+import { tweet } from 'utils/v2/tweet'
+
+import LaunchProjectPayerModal from './LaunchProjectPayerModal'
+
+export default function NewDeployModal({
+  visible,
+  onClose,
+}: {
+  visible: boolean
+  onClose: VoidFunction
+}) {
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+  const projectURL = window.location.origin + useLocation().pathname
```

Comment:
is `origin` returned in the `useLocation()` hook?

Reply:
Nah it's not